hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -183,31 +181,11 @@ def get_callbacks(self) -> List[Callback]:
     def get_checkpoint_to_test(self) -> Path:
         """
         Returns the full path to a checkpoint file that was found to be best during training, whatever criterion
-        was applied there. This is necessary since for some models the checkpoint is in a subfolder of the checkpoint
-        folder.
+        was applied there. Note that the checkpoint file might not (yet) exist, for example in the case of
+        preempted jobs.
         """
-        # absolute path is required for registering the model.
```

Comment:
do you know why we had to do this "not so nice" workaround before?

Reply:
I honestly have no idea. I have removed it because it also is in the way of the new functionality. If it turns out that this old complex logic is indeed needed, I'll have to add it back and re-design.